scripts/tests/test_user_summary.py

```diff
@@ -1,28 +1,17 @@
 from datetime import datetime, timedelta
+from django.utils import timezone
 from nose.tools import *  # noqa
-from framework.mongo import database
-from framework.transactions.context import TokuTransaction
 
-from website.models import User
+from osf.models import OSFUser
 from tests.base import OsfTestCase
-from tests.factories import AuthUserFactory, NodeLogFactory
+from osf_tests.factories import AuthUserFactory, NodeLogFactory
 from scripts.analytics.user_summary import UserSummary, LOG_THRESHOLD
 
 
-def modify_user_dates_in_mongo(new_date):
-    with TokuTransaction():
-        for user in database.user.find():
-            database['user'].find_and_modify(
-                {'_id': user['_id']},
-                {'$set': {
-                    'date_registered': new_date
-                }}
-            )
-
 class TestUserCount(OsfTestCase):
     def setUp(self):
-        self.yesterday = datetime.today() - timedelta(1)
-        self.a_while_ago = datetime.today() - timedelta(2)
+        self.yesterday = timezone.now() - timedelta(1)
```

Comment:
datetime wasn't including the timezone information! So I was getting "can't compare naive and aware datetimes" when trying to do date math.